plugin/build_baron.go

```diff
@@ -105,3 +118,75 @@ func (bbp *BuildBaronPlugin) GetPanelConfig() (*PanelConfig, error) {
 		},
 	}, nil
 }
+
+func IsWebhookConfigured(project string, version string) (evergreen.WebHook, bool) {
```

Comment:
Now that this function is a bit more complicated, it would be helpful to add a comment around what version is for and the lookup hierarchy